docs/tutorials/blog.md

```diff
@@ -57,6 +57,8 @@ npm run dev
 
 💿 Open up [http://localhost:3000][http-localhost-3000], the app should be running.
 
+<docs-info>Note, if you encounter an error that says `Error: Invariant failed: SESSION_SECRET must be set`. Try adding a new .env file with the same content as the .env.example file.</docs-info>
```

Comment:
Hi @GaneshSrambikal 👋🏼 

I just followed through the tutorial, and the `.env` should have been created for you by the setup script. When running:
```
npx create-remix@latest --template remix-run/indie-stack blog-tutorial
```

You get prompted to run it:
```
  init   This template has a remix.init script. Do you want to run it?
         Yes / No
```

If you ever missed that, the blog tutorial also says to follow along the README in the generated project, which in turn also instruct you to run the setup script.

I'm all for adding another heads up in the blog tutorial, as you suggest it, but instead of warning about potential error messages, I'd rather phrase like: 

```
Don't forget to run the setup script manually (`npm run setup`) if you opted-out of running it during the installation.
```

What do you think?